### ISSUE TITLE
URL's shouldn't give CodeReview false positives when they're not pointing to CodeReview

### ISSUE THREAD

**Issue by retailcoder** [skiwi2/GithubHookSEChatService#62]
- [ ] Determine if a comment contains a URL (regex?)
- [ ] Determine if rest of comment contains a match. If not...
- [ ] Determine if URL is pointing to `codereview.stackexchange.com` - any other domain/subdomain or part of a URL that doesn't match that, is a false positive.

**Comment by Zomis** [comment]
Erhm....! `codereview.stackexchange.com/help/on-topic`

**Comment by retailcoder** [comment]
@Zomis what's the domain/subdomain of that url again?

> `codereview.stackexchange.com`

hmm maybe I need to fine-tune the way I word specs... still :+1: that bot is pure awesome!

**Comment by retailcoder** [comment]
Oh wow, I think I just pinged someone named Duga!

**Comment by Zomis** [comment]
This problem also exist for the "Programmers" website, and it is a really difficult problem. For programmers.stackexchange I scan for keywords such as "better fit", "place to ask", "belong", etc. and I have still not been able to eliminate false positives. While the same thing could be done for Code Review, we don't have as many false positives.

**Comment by Vogel612** [comment]
Another idea might be to ignore references to specific questions. There is no necessitiy to catch things like:

    [closely related](http://codereview.stackexchange.com/q/45918/489)

**Comment by Zomis** [comment]
Yup. I'd like to come up with a nice way of filter things here. It is also important to remember that a theoretical comment such as:

    This question is a better fit for Code Review,
     see [this related question over there as well](http://codereview.stackexchange.com/q/45918/489)

Such comments should still be posted.
